src/components/navigation/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useMemo } from 'react'
+import { fileAbsolute } from 'paths.macro'
+import { getStoryTitle } from '../../../storybook/get-story-title'
+import { StoryFn } from '../../../storybook/utils/storybook-types'
+import { Box, Screen, Card } from '../../structure'
+import { TextContainer, Heading, BodyText } from '../../text'
+import { LOREM_IPSUM } from '../../../storybook/utils/constants'
+import { PhoneScreen } from '../../../storybook/decorators/PhoneScreen'
+import { Icon } from '../../images-and-icons'
+import { Tabs, TabsProps } from './Tabs'
+
+export default {
+  title: getStoryTitle(fileAbsolute),
+  component: Tabs,
+  decorators: [PhoneScreen],
+}
+
+export const Basic: StoryFn<TabsProps> = props => {
+  const [selected, setSelected] = useState(props.selected ?? 0)
+
+  return (
+    <Screen>
+      <Tabs {...props} selected={selected} onSelect={setSelected}>
+        <Screen.Content>
+          <Box padding="medium">
+            <TextContainer>
+              <Heading>{props.tabs[selected].label}</Heading>
+              <BodyText>{LOREM_IPSUM}</BodyText>
+            </TextContainer>
+          </Box>
+        </Screen.Content>
+      </Tabs>
+    </Screen>
+  )
+}
+
+Basic.args = {
+  tabs: [
+    { label: 'Developers', id: 'devs' },
+    { label: 'Designers', id: 'designers' },
+  ],
+  selected: 0,
+}
+
+Basic.argTypes = {
+  children: { control: null },
+}
+
+export const DifferentTabsContent: React.FC = () => {
+  const [selected, setSelected] = useState(0)
+
+  const tabContent = useMemo(() => {
+    switch (selected) {
+      case 0:
+        return <BodyText>Developers like things simple.</BodyText>
+      case 1:
+        return (
+          <Card title="Designers" sectioned>
+            <Box horizontal space="small">
+              <Icon name="theme" color="accent" />
+              <BodyText>Designers like beautiful things!</BodyText>
+            </Box>
+          </Card>
+        )
+      default:
+        throw new Error('Invalid tab')
+    }
+  }, [selected])
```

Comment:
I think it'd be better to just have a basic `if` - or `switch` with `default` - directly inside `Screen.Content`.

Readers will have different experience levels with React so they might not be familiar with more advance concepts as `useMemo`. Plus, it might not be the style they're using.

I'd aim to keep the code samples as simple and neutral as possible! :)

Reply:
Yes of course! Replaced with ternary if without `useMemo`